cmd/ts/metrics.go

```diff
@@ -0,0 +1,78 @@
+package ts
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/loft-sh/devpod/pkg/tailscale"
+	"github.com/spf13/cobra"
+)
+
+type MetricsCmd struct {
+	*TsNetFlags
+}
+
+type TsNetFlags struct {
+	AccessKey       string
+	PlatformHost    string
+	NetworkHostname string
+}
+
+func (flags *TsNetFlags) ParseFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&flags.AccessKey, "access-key", "", "")
+	cmd.Flags().StringVar(&flags.PlatformHost, "host", "", "")
+	cmd.Flags().StringVar(&flags.NetworkHostname, "hostname", "", "")
+}
+
+func NewMetricsCmd() *cobra.Command {
+	cmd := &MetricsCmd{&TsNetFlags{}}
+	var metricsCmd = &cobra.Command{
+		Use:   "metrics print",
```

Comment:
can we use a single string with no white spaces?

Reply:
I could, I was trying to mirror the tailscale CLI as is so that you could literally alias `tailscale` for `devpod ts` so the tailscale docs remain the same. Also we then have the potential to steal other scripts etc in the future if they expect the tailscale CLI. What do you think? https://tailscale.com/kb/1080/cli#metrics. Happy either way though as I can see advantages to us changing the code for our specific debugging tasks as well.

Reply:
sorry forgot to check this PR 🤦 yeah sounds good, if it's mirroring upstream then lets change as little as possible